lib/components/TokenDetailsCard.jsx

```diff
@@ -1,338 +1,252 @@
+    tokenDetailsDescription = (
+      <>
+        The yield source at the provided address must implement the Yield Source Interface. An ERC20
+        token will be accessible from the address supplied which defines what a user deposits to
+        join the prize pool.
+      </>
+    )
   }
-  tokenDetailsDescription +=
-    ' When a user deposits, they will receive a token back representing their deposit and chance to win. The name and symbol of this ticket token can be customized in “Advanced Settings”.'
 
-  return (
-    <Card>
-      <InputLabel primary={label} description={tokenDetailsDescription} className='mb-4'>
-        <PrizePoolInputs
-          prizePoolType={prizePoolType}
-          // Compound Prize Pool
-          updateCToken={updateCToken}
-          cToken={cToken}
-          // Staked Prize Pool
-          stakedTokenAddress={stakedTokenAddress}
-          stakedTokenData={stakedTokenData}
-          setStakedTokenAddress={setStakedTokenAddress}
-          setStakedTokenData={setStakedTokenData}
-          updateTicketLabels={updateTicketLabels}
-          // Yield Prize Pool
-          yieldSourceAddress={yieldSourceAddress}
-          setYieldSourceAddress={setYieldSourceAddress}
-          yieldSourceData={yieldSourceData}
-          setYieldSourceData={setYieldSourceData}
-        />
-      </InputLabel>
+  const tokenDetailsDescriptionSecondary = (
+    <>
+      When a user deposits, they will receive a token back representing their deposit and chance to
+      win. The name and symbol of this ticket token can be customized below.
+    </>
+  )
 
-      <InputLabel
-        secondary='Pool Ticket'
-        description={`Provide a name and ticker symbol for the ERC20 token that will be created and used as the pool's tickets.`}
-      >
-        <div className='flex flex-col sm:flex-row sm:mb-4'>
-          <TextInputGroup
-            containerClassName='w-full sm:w-1/2 sm:mr-4'
-            id='_ticketName'
-            label={`Ticket's name`}
-            placeholder='(eg. PT Compound Dai Ticket)'
-            required
-            onChange={(e) => {
-              setUserChangedTicketName(true)
-              setTicketName(e.target.value)
-            }}
-            value={ticketName}
-          />
+  const isYieldPool =
+    prizePool.type === PRIZE_POOL_TYPE.customYield || prizePool.type === PRIZE_POOL_TYPE.compound
 
-          <TextInputGroup
-            id='_ticketSymbol'
-            containerClassName='w-full sm:w-1/2 sm:ml-4'
-            label='Ticket Ticker'
-            placeholder='(eg. PCDAI)'
-            required
-            maxLength='5'
-            onChange={(e) => {
-              setUserChangedTicketSymbol(true)
-              setTicketSymbol(e.target.value)
-            }}
-            value={ticketSymbol}
+  return (
+    <div style={{ marginTop: -50 }}>
+      <div className='bg-card pt-6 sm:pt-0'>
+        <Card>
+          {!isYieldPool && <StakeDisplay prizePool={prizePool} />}
```

Comment:
The tooltips that show up in the first paragraph could be duplicated here for double the clarity on what the types of pools are (mostly just that "Stake" requires a prize manually added)

Reply:
Now might also be a great time to change the name of "Stake" pools too. I don't think "stake" is very clear. "Owner Funded Pools", "Funded Pools" or even "Manual Prize" makes more sense to me at a first glance. "Stake" does line up with all of our other docs though.